frontend/src/data/variables/VariableProvider.ts

```diff
@@ -71,6 +71,25 @@ abstract class VariableProvider {
       .resetIndex();
   }
 
+  // If County fips exists and is 'Short'
```

Comment:
'Short' is kinda confusing... Maybe just change comment to say "convert 3 character county_fips to 5 characters by prepending state fips, if present"

Although, I don't understand why the backend can't just provide the correct format?